framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -2307,6 +2307,7 @@ public ParameterizedExecutableType methodFromUse(
       adaptGetClassReturnTypeToReceiver(methodType, receiverType, tree);
     }
 
+    methodType.setReturnType(applyCaptureConversion(methodType.getReturnType()));
```

Comment:
I'm not sure if this is the right place to applyCaptureConversion.  I need to try this change in my 979 branch.